docs/rst/knora-api-server/api_v1/create-a-mapping.rst

```diff
@@ -0,0 +1,358 @@
+ - ``<strong>`` -> ``standoff:StandoffBoldTag``
+ - ``<u>`` -> ``standoff:StandoffUnderlineTag``
+ - ``<sub>`` -> ``standoff:StandoffSubscriptTag``
+ - ``<sup>`` -> ``standoff:StandoffSuperscriptTag``
+ - ``<strike>`` -> ``standoff:StandoffStrikeTag``
+ - ``<a href="URL">`` -> ``knora-base:StandoffUriTag``
+ - ``<a class="salsah-link" href="Knora IRI">`` -> ``knora-base:StandoffLinkTag``
+ - ``<br>`` -> ``standoff:StandoffBrTag``
+
+The HTML produced by CKEditor is wrapped in an XML doctype and a pair of root tags ``<text>...</text>`` and then sent to Knora. The XML sent to the GUI by Knora is unwrapped accordingly (see ``jquery.htmleditor.js``).
+Although the GUI supports HTML5, it is treated as if it was XHTML in strict XML notation.
+
+*************************
+Creating a custom Mapping
+*************************
+
+The Knora standard mapping only supports a few HTML tags. In order to submit more complex XML markup to Knora, a custom mapping has to be created first.
+Basically, a mapping expresses the relations between XML elements and attributes and their corresponding standoff classes and properties.
+The relations expressed in a mapping are one-to-one relations, so the XML can be recreated from the data in RDF. However, since HTML offers a very limited set of elements, Knora mappings support the combination of element names
+and classes. In this way, the same element can be used several times in combination with another classname (please note that ``<a>`` without a class is a mere hyperlink whereas ``<a class="salsah-link">`` is an internal link/standoff link).
+
+----------------------------
+Basic Structure of a Mapping
+----------------------------
+
+The mapping is written in XML itself (for a formal description, see ``webapi/src/resources/mappingXMLToStandoff.xsd``). It has the following structure (the indentation corresponds to the nesting in XML):
+
+- ``<mapping>``: the root element
+    - ``<mappingElement>``: an element of the mapping (at least one)
+       - ``<tag>``: information about the XML element that is mapped to a standoff class
+           - ``<name>``: name of the XML element
+           - ``<class>``: value of the class attribute of the XML element, if any. If the element has no class attribute, the keyword ``noClass`` has to be used.
+           - ``<namespace>``: the namespace the XML element belongs to, if any. If the element does not belong to a namespace, the keyword ``noNamespace`` has to be used.
+           - ``<separator>``: a Boolean value indicating if a separator is required for this element. Once an XML document is converted to RDF-standoff the markup is stripped from the text, possibly leading to continuous text that has been separated by tags before. For structural tags like paragraphs etc., ``<separator>`` can be set to ``true`` in which case a special separator is inserted in the the text in the RDF representation. In this way, words stay separated and are represented in the fulltext index as such.
```

Comment:
I think it would be clearer to call this something like `<separatesWords>`. Then the description could say "a Boolean value indicating whether this tag separates words in the text".